eks-anywhere-common/Testers/Nirmata/kyverno-testjob.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: kyverno-tester
+  namespace: kyverno
+spec:
+  template:
+    spec:
+      serviceAccountName: 'kyverno'
+      containers:
+        - name: job
+          image: 'alpine/k8s:1.26.2'
+          imagePullPolicy: Always
+          command:
+            - /bin/bash
+          args:
+            - -c
+            - >-
+              echo 1. Checking readiness probe...
+              kyverno_pod="...";
```

Comment:
This job is not functionally validating Kyverno. This is like a technical test.

Reply:
Created the cronjob and added check for policy creation and status. Pls confirm if this looks OK.